Documentation~/workflow-edit-tasks.md

```diff
@@ -28,28 +30,30 @@ This page describes some of the most common ways you can use the ProBuilder tool
-  - Press **Backspace** to delete the insets. 
+  - Select the wall's two faces (back and front), then **Shift**+**Scale** to create an [inset](Face_Inset.md) horizontally, and **Scale** vertically to make it an even border.
+  - Press **Backspace** to delete the insets.
 
-  On the other hand, if you are working on an uneven or multi-sided wall, such as a castle or church wall, this strategy is preferable: 
+  On the other hand, if you are working on an uneven or multi-sided wall, such as a castle or church wall, this strategy is preferable:
 
-  * Use the [Insert Edge Loop](Edge_InsertLoop.md) tool to create two vertical edges. Do the same on the horizontal plane (two for a window, one for a door). 
-  * Adjust the loops so that the resulting hole is the right size and location for a window or a door. 
+  * Use the [Insert Edge Loop](Edge_InsertLoop.md) action to create two vertical edges. Do the same on the horizontal plane (two for a window, one for a door).
+  * Adjust the loops so that the resulting hole is the right size and location for a window or a door.
   * Select the face of the hole and press **Backspace**. If necessary, from the other side of the wall, [delete the face](Face_Delete.md) on the other side.  
 
   Whichever strategy you start with, you need to weld the newly exposed edges and vertices together:
 
-  * Select all the edges on one side only of the hole and use **Shift**+**Translate** until they connect up with the edge on the other side. 
+  * Select all the edges on one side only of the hole and use **Shift**+**Translate** until they connect up with the edge on the other side.
   * Then [weld the vertices](Vert_Weld.md) together where the edges meet.
 
 - **Add a door**: follow the *Make a hole* procedure, but to start, create a [Door shape](Door.md). Fit the hole to match, [merge the two objects](Object_Merge.md), and then [weld the door to the frame](Vert_Weld.md) you created.
 
-- **Make a tunnel with normals on both the inside and outside**: duplicate the tunnel object and scale it slightly so that you [bridge the edges](Edge_Bridge.md). Next, [flip the normals](Object_FlipNormals.md) on the smaller one. 
+- **Make a tunnel with normals on both the inside and outside**: duplicate the tunnel object and scale it slightly so that you [bridge the edges](Edge_Bridge.md). Next, [flip the normals](Object_FlipNormals.md) on the smaller one.
 
 - **Make a building with towers**: start with a [cube](Cube.md) and [inset](Face_Inset.md) on the top face, then [extrude](Face_Extrude.md) upward. Repeat this as many times as you like. You could also [subdivide the top face](Face_Subdivide.md) to create multiple extrusions.
 
+<span style="color:blue">@DEVNOTE: Do you have some good examples for using the Cut tool to make either objects or buildings for this section here? </span>
+
```

Comment:
It's a really good tool to make custom cuts and broke flat surfaces in smaller ones. 
But @gabrielw-us will have more examples on this I guess :)